### ISSUE TITLE
Where to download YOLO_NAS python repo ONLY?

### ISSUE THREAD

**Issue by ToniButland1998** [Deci-AI/super-gradients#1040]
### 💡 Your Question

Where to download python repo and pre-trained models only? I wanna download the YOLO_NAS python repos only, so that I can easily train on my dataset. Thanks.

**Comment by ofrimasad** [comment]
Hi @ToniButland1998.
you came to the right place.
This repo is the YOLO-NAS repo.
YOLO-NAS is part of SuperGradients training library.
feel free to clone the git repo or [install using pip](https://docs.deci.ai/super-gradients/documentation/source/installation.html).
